torchvision/ops/misc.py

```diff
@@ -142,15 +143,12 @@ def _load_from_state_dict(self, state_dict, prefix, local_metadata, strict,
             missing_keys, unexpected_keys, error_msgs)
 
     def forward(self, x):
-        # move reshapes to the beginning
-        # to make it fuser-friendly
-        w = self.weight.reshape(1, -1, 1, 1)
-        b = self.bias.reshape(1, -1, 1, 1)
-        rv = self.running_var.reshape(1, -1, 1, 1)
-        rm = self.running_mean.reshape(1, -1, 1, 1)
-        scale = w * rv.rsqrt()
-        bias = b - rm * scale
-        return x * scale + bias
+        # Scaling factor
+        scale = self.weight * (self.running_var + self.eps).rsqrt()
+        # Bias
+        bias = (self.bias - self.running_mean * scale)
+
+        return x * scale.view(1, -1, 1, 1) + bias.view(1, -1, 1, 1)
```

Comment:
In general, I prefer to use `reshape` instead of `view`, because it's safer: `view` can raise errors if the tensor is not contiguous, while `reshape` makes it contiguous beforehand. Plus, `reshape` is smart so that it doesn't make unnecessary copies, so for me it is strictly better than `view`.